scripts/config/create-config.ts

```diff
@@ -0,0 +1,221 @@
+import { join } from 'path';
+import { ethers } from 'ethers';
+import { encode } from '@api3/airnode-abi';
+import { AirnodeRrpAddresses } from '@api3/airnode-protocol';
+import { readOperationsRepository } from '@api3/operations/dist/utils/read-operations';
+import { ChainOptions } from '@api3/airnode-node';
+import {
+  readConfigurationData,
+  runAndHandleErrors,
+  writeJsonFile,
+  getDapiServerInterface,
+  sanitiseFilename,
+} from './utils';
+import { Config, ChainConfig, NodeSettings, Subscriptions, Triggers, Templates, Endpoints } from '../../src';
+
+const main = async () => {
+  const operationsRepository = readOperationsRepository();
+  const ConfigurationData = readConfigurationData();
+
+  const { airnode, xpub } = Object.values(ConfigurationData.beacons)[0] as any;
+
+  const apiChains = [
+    ...new Set(Object.values(ConfigurationData.beacons).flatMap((beacon: any) => Object.keys(beacon.chains))),
+  ];
+
+  const chains: ChainConfig[] = apiChains.map((chainName) => {
+    const chainId = parseInt(operationsRepository.chains[chainName].id);
+    const contracts = {
+      AirnodeRrp: AirnodeRrpAddresses[chainId] || '',
+      RrpBeaconServer: operationsRepository.chains[chainName].contracts.RrpBeaconServer || '',
+      DapiServer: operationsRepository.chains[chainName].contracts.DapiServer || '',
+    };
+    const options: ChainOptions = {
+      fulfillmentGasLimit: 500000,
+      gasPriceOracle: [
+        {
+          gasPriceStrategy: 'constantGasPrice',
+          gasPrice: {
+            value: 10,
+            unit: 'gwei',
+          },
+        } as const,
+      ],
+    };
+    const providers = {
+      [`provider_${sanitiseFilename(chainName).replace(/\-/g, '_')}`]: {
+        url: `\${${sanitiseFilename(chainName).replace(/\-/g, '_')}_PROVIDER_URL}`.toUpperCase(),
+      },
+    };
+
+    return {
+      contracts,
+      id: `${chainId}`,
+      providers,
+      type: 'evm' as const,
```

Comment:
Do you mind explaining why we need this casts `as const` here and there?

Reply:
The whole return can even be a `ChainConfig` object type

Reply:
yeah, I realized it doesn't need to be const. Fixed